From b19c088ce95e8ca6ff1102c36d91b1db937e3a3a Mon Sep 17 00:00:00 2001
From: d2weber <d2weber@noreply.codeberg.org>
Date: Wed, 5 Nov 2025 14:02:41 +0100
Subject: [PATCH] Fix undefined reference to addIdToMemberlist (#106)

This fixes the following runtime error of deltatouch when built with CLICK_MODE=OFF
```
qrc:/Main.qml:28: Warning: qrc:/Main.qml:28:1: plugin cannot be loaded for module "DeltaHandler": Cannot load library /nix/store/hyjwrfl00n3cf0v5hfmp4r5hffvqnhcm-deltatouch-2.22.0/lib/qt5/plugins/DeltaHandler/libDeltaHandler.so: (/nix/store/hyjwrfl00n3cf0v5hfmp4r5hffvqnhcm-deltatouch-2.22.0/lib/qt5/plugins/DeltaHandler/libDeltaHandler.so: undefined symbol: _ZN13ContactsModel17addIdToMemberlistEj)
     import DeltaHandler 1.0
     ^
```

Reviewed-on: https://codeberg.org/lk108/deltatouch/pulls/106
Co-authored-by: d2weber <d2weber@noreply.codeberg.org>
Co-committed-by: d2weber <d2weber@noreply.codeberg.org>
---
 src/plugins/DeltaHandler/contactsmodel.h | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/plugins/DeltaHandler/contactsmodel.h b/src/plugins/DeltaHandler/contactsmodel.h
index f2596a0..bf38034 100644
--- a/src/plugins/DeltaHandler/contactsmodel.h
+++ b/src/plugins/DeltaHandler/contactsmodel.h
@@ -69,7 +69,6 @@ public slots:
 
     // Used in the page to add members to a group
     void startChatWithIndex(int myindex);
-    void addIdToMemberlist(uint32_t contactId);
 
     void updateContacts(uint32_t accId, int contactId);
 
